### ISSUE TITLE
Incorrect privacy manifest

### ISSUE THREAD

**Issue by yliu342** [SocketMobile/swift-package-capturesdk#19]
@cyrille-socket When submit app for review we get warnings regarding `ITMS-91053: Missing API declaration`. After looking at the app bundle, I saw `Capture SDK` and `SwiftDecoder` are the one that uses those required API.

Using version Capture SDK 1.8.57 and SwiftDecoder 5.8.14 I can see both SDK are statically linked to our app.
Capture SDK has a PrivacyInfo.xcprivacy file within the CaptureSDK.framework. But this file doesn't declare any use of those required api meaning the `NSFileCreationDate`.
SwiftDecoder SDK doesn't have a PrivacyInfo.xcprivacy file at all.

Can we address this with a release or at least provide some guidance what the SDK uses those API for?
NSPrivacyAccessedAPICategoryFileTimestamp
NSPrivacyAccessedAPICategoryDiskSpace
NSPrivacyAccessedAPICategorySystemBootTime

```
SwiftDecoder.framework/SwiftDecoder: fstat, mach_absolute_time, NSUserDefaults, stat
CaptureSDK.framework/CaptureSDK: NSFileCreationDate
```

**Comment by ios-dev-socket** [comment]
Hi @yliu342 . We just released the version `1.9.30` of our SDK on our Developer Portal which contains the Privacy Policy for SwiftDecoder.

**Comment by cyrille-socket** [comment]
This issue has been resolved with version `1.9.30`.
